lib/operations.rb

```diff
@@ -20,9 +21,26 @@
 require "operations/form/attribute"
 require "operations/form/builder"
 
-# Dry::Schema.load_extensions(:monads)
-# Dry::Validation.load_extensions(:monads)
+# The root gem module
+module Operations
+  class Error < StandardError
+  end
 
-# Your code goes here...
-class Operations::Error < StandardError
+  DEFAULT_ERROR_REPORTER = lambda do |message, payload|
```

Comment:
shouldn't it be in configuration ?

Reply:
Yes and it is below

Reply:
I mean, don't we want to expose default instance of configuration on onfiguration class ?

Reply:
but I did it, there is default_config which does exactly this